backends/gstreamer/src/media_stream.rs

```diff
@@ -1,6 +1,6 @@
 use glib::ObjectExt;
-use gst::{self, BinExtManual, ElementExt, GObjectExtManualGst};
-use servo_media_webrtc::MediaStream;
+use gst::{self, BinExt, BinExtManual, ElementExt, GObjectExtManualGst};
```

Comment:
I'd recommend simply importing `glib::prelude::*` and `gst::prelude::*` instead of the traits manually btw